src/DataTable/TableRow.js

```diff
@@ -64,60 +69,92 @@ const TableRowStyle = styled.div`
   ${props => props.extendedRowStyle};
 `;
 
-const TableRow = memo(({
-  id,
-  keyField,
-  columns,
-  row,
-  onRowClicked,
-  onRowDoubleClicked,
-  selectableRows,
-  expandableRows,
-  striped,
-  highlightOnHover,
-  pointerOnHover,
-  dense,
-  expandableRowsComponent,
-  expandableDisabledField,
-  defaultExpanded,
-  expandOnRowClicked,
-  expandOnRowDoubleClicked,
-  conditionalRowStyles,
-}) => {
-  const [expanded, setExpanded] = useState(defaultExpanded);
-  const handleExpanded = useCallback(() => {
-    setExpanded(!expanded);
-  }, [expanded]);
-
-  const disableRowClick = row[expandableDisabledField] || false;
-  const showPointer = pointerOnHover || (expandableRows && (expandOnRowClicked || expandOnRowDoubleClicked));
-
-  const handleRowClick = useCallback(e => {
-    // use event delegation allow events to propagate only when the element with data-tag ___react-data-table-allow-propagation___ is present
-    if (e.target && e.target.getAttribute('data-tag') === STOP_PROP_TAG) {
-      onRowClicked(row, e);
-
-      if (!disableRowClick && expandableRows && expandOnRowClicked) {
-        handleExpanded();
-      }
-    }
-  }, [disableRowClick, expandOnRowClicked, expandableRows, handleExpanded, onRowClicked, row]);
-
-  const handleRowDoubleClick = useCallback(e => {
-    if (e.target && e.target.getAttribute('data-tag') === STOP_PROP_TAG) {
-      onRowDoubleClicked(row, e);
-      if (!disableRowClick && expandableRows && expandOnRowDoubleClicked) {
-        handleExpanded();
-      }
-    }
-  }, [disableRowClick, expandOnRowDoubleClicked, expandableRows, handleExpanded, onRowDoubleClicked, row]);
-
-  const extendedRowStyle = getConditionalStyle(row, conditionalRowStyles);
-
-  return (
-    <>
+const TableRow = memo(
+  ({
```

Comment:
We should preserve the original formatting to keep not only to keep the diff clean but to be consistent with other components